### PR TITLE
Use `adoptopenjdk:8` as a base image in Dockerfile used for CI [BT-93]

### PR DESCRIPTION
**Pull request by grsterin**

### CODE REVIEW

File: src/ci/docker-compose/cromwell-test/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM us.gcr.io/broad-dsp-gcr-public/base/jre:8-debian
+FROM adoptopenjdk:8
```
Comment (mcovarr) [inline src/ci/docker-compose/cromwell-test/Dockerfile:1]: Before the recent change this had been `openjdk:8`; is there a reason behind this new choice of value?

Comment (grsterin) [reply src/ci/docker-compose/cromwell-test/Dockerfile:1]: I just saw that in some other file Chris recently changed `openjdk:8` to `adoptopenjdk:8`. I think this is something Trivy-related, but I don't know enough context around this. @cjllanwarne should we use  `openjdk:8` or `adoptopenjdk:8` here?

Comment (grsterin) [reply src/ci/docker-compose/cromwell-test/Dockerfile:1]: Confirmed here: https://broadinstitute.slack.com/archives/GHYJZ2ZE0/p1611084632053600

**Review by kshakir — APPROVED** [review]
I'm happy assuming you can quote an example where you ran Jenkins on this branch and it's happy?

**Review by mcovarr — APPROVED** [review]

**Comment by grsterin** [comment]
> I'm happy assuming you can quote an example where you ran Jenkins on this branch and it's happy?

https://gotc-jenkins.dsp-techops.broadinstitute.org/view/cromwell-cron/job/cromwell-cron-aws/890/console

**Review by breilly2 — APPROVED** [review]

**Pull request merged**